### DISCUSSION TITLE
Quick poll question

### DISCUSSION THREAD

**Discussion by zacharytomlinson** [zacharytomlinson/saspy#38] [Q&A] [answered]
@well-it-wasnt-me  Hey bud, do you happen to know exactly what this poll is doing around line 2256? 

`    def aft_jp(self, money, amount=1, lock_timeout=0, games=None):` 

I am doing a bit of re-org to allow for a separate RTClient (real-time client) so we don't end up having one SAS God class that is 15 thousand lines of code and wasn't sure whether some of the attributes it assigns to the SAS scope should actuallly be owned by SAS or by the functions scope and managed by the consumer of the SAS client.

Some examples of the changes I am making is to things like having a separate AFT namespace that has the associated classes living inside of it and the AFT module will have attributes like the registration key stored within it instead of the SAS class. If we include all sub-functionality of the SAS protocol and each of their respective states and data all within SAS it is going to be an unreadable behemoth.

**Comment by well-it-wasnt-me** [comment]
Give me 20 min and I'll tell you

**Comment by well-it-wasnt-me** [comment, marked as answer]
Per documentation 
> "Transfer bonus jackpot win amount from host to gaming machine (force attendant pay lockup)"
